package.json

```diff
@@ -19,12 +27,10 @@
     "cookbook": "cd polyfill && npm install && npm run cookbook && cd ..",
     "test262": "cd polyfill && npm install && npm run test262",
     "lint": "eslint . --ext js,mjs",
-    "build:prepare": "mkdirp out && mkdirp out/docs",
-    "build:polyfill": "cd polyfill && npm install && npm run build && cd ..",
-    "build:javascript": "npm run build:polyfill && cd docs && npm install && npm run build:javascript && cd .. && cp docs/*.js docs/*.js.map out/docs/",
-    "build:docs": "cd docs && npm install && npm run build:html && cd .. && cp docs/*.html docs/*.css out/docs/",
+    "build:javascript": "cd polyfill && npm install && npm run build && cd .. && rollup -m -c rollup.config.ejs",
```

Comment:
```suggestion
    "prebuild:javascript": "cd polyfill && npm install && npm run build && cd ..",
    "build:javascript": "rollup -m -c rollup.config.ejs",
```

Reply:
ok, also removing extraneous `&& cd ...` in follow-on commit b7ae15d56